packages/design-tokens/src/tokens/functional/components/subdomain-nav-bar/colors.js

```diff
@@ -3,8 +3,8 @@ module.exports = {
     SubdomainNavBar: {
       canvas: {
         default: {
-          value: 'hsl(var(--base-color-scale-white-0-hsl) / 50%)',
-          dark: 'hsl(var(--base-color-scale-gray-9-hsl) / 50%)',
+          value: '#ffffff80',
+          dark: '#0d111780',
```

Comment:
Could we add a TODO here to handle this via a StyleDictionary transformer? If the gray-9 value changes, it won't get reflected here, so would be better to handle this via interpolation in the long term. An annotation here would help remind us to do this (and also clarify this is gray-9 at 50%)

Reply:
Remember this is temporary, so the new build process already handles this. It looks something like this:

```
hover: {
          $value: '{base.color.gray.2}',
          $type: 'color'
          },
          alpha: 0.2,
        },
```

Good idea to annotate 👍 will do that to make it easier to upgrade later